source/presentations.rst

```diff
@@ -25,7 +25,38 @@ Presentations & Articles
   * `Python packaging simplified, for end users, app developers, and open source
     contributors <https://us.pycon.org/2014/schedule/presentation/219>`_
 
+* SciPy, July 2016
+
+  * `Reinventing the whl: New Developments in the Upstream Python Packaging Ecosystem
+    <http://pyvideo.org/scipy-2016/reinventing-the-whl-new-developments-in-the-upstream-python-packaging-ecosystem-scipy-2016-nath.html>`_
+
+* PyCon FR, October 2016
+
+  * `Warehouse - the future of PyPI
+    <http://pyvideo.org/pycon-fr-2016/warehouse-the-future-of-pypi.html>`_
+
+* PyCon US, May 2017
+
+  * `Share Your Code! Python Packaging Without Complication
+    <https://daveops.com/talks/pycon-2017-share-your-code-python-packaging-without-complication/>`_
+
+* PyGotham, October 2017
+
+  * `Running Vintage Software: PyPI's Aging Codebase
+    <http://pyvideo.org/pygotham-2017/running-vintage-software-pypis-aging-codebase.html>`_
+
+* North Bay Python, December 2017
+
+  * `Running Vintage Software: PyPI's Aging Codebase
+    <https://2017.northbaypython.org/schedule/presentation/5/>`_
+
```

Comment:
I think Mahmoud Hashemi's talk "The Packaging Gradient" might be worthwhile to add here: https://www.youtube.com/watch?v=iLVNWfPWAC8

Also the corresponding essay: http://sedimental.org/the_packaging_gradient.html

Reply:
Thanks! Adding that and adding further history -- will change PR title back to remove "WIP:" prefix and ask for re-review when it's ready.